tests/acceptance/settings/settings-account-test.js

```diff
@@ -225,18 +214,28 @@ test('visit ' + settingsAccountUrl + ' change email and errors', function(){
   });
 
   andThen(function(){
-    fillIn(emailInput(), newEmail);
-    click(changeEmailButton());
+    fillInput('email', newEmail);
+    clickButton('Change email');
   });
 
   andThen(function(){
-    fillIn(currentPasswordInput(), currentPassword);
-    click(changeEmailButton());
+    fillInput('current-password', currentPassword);
+    clickButton('Change email');
   });
 
   andThen(function(){
-    var error = find('.alert');
+    let error = find('.alert');
     ok(error.length, 'shows error div');
     ok(error.text().indexOf('Invalid password'), 'shows error message');
   });
+
+  andThen( () => {
+    visit(settingsProfileUrl); // go away
+    visit(settingsAccountUrl); // come back
+  });
```

Comment:
Yeah, I think this improves test legibility because it groups the leave-and-return step from the error-checking assertions above it. If that seems unneeded these can go in the above andThen